app-misc/cdargs/cdargs-2.1.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson elisp-common
+
+DESCRIPTION="Bookmarks and browser for the shell builtin cd command"
+HOMEPAGE="https://www.skamphausen.de/cgi-bin/ska/CDargs https://github.com/cbxbiker61/cdargs"
+SRC_URI="https://github.com/cbxbiker61/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
should be `GPL-2+`, see:
https://github.com/cbxbiker61/cdargs/blob/master/src/cdargs.cc#L10C1-L13C43